quickwit/quickwit-ui/src/views/SearchView.tsx

```diff
@@ -66,8 +69,8 @@ function SearchView() {
     setSearchRequest(updatedSearchRequest);
     setQueryRunning(true);
     setSearchError(null);
+    navigate('/search?' + toUrlSearchRequestParams(updatedSearchRequest).toString());
```

Comment:
Update the URL once the "run" action is triggered, was bad to update it only if search request was successful.